Quality.md

```diff
@@ -17,13 +17,13 @@
 ### I ensure Octopus teams can ship with confidence
 
 - I am an advocate for quality.
```

Comment:
I am an advocate for quality and collaborate with the team to ensure each release is of better quality than we had before.